### PR TITLE
Styling improvements to the Mortgage tools

### PR DESCRIPTION
**Pull request by slaywell**
Adding spacing to buttons on mobile view, e.g. Stamp Duty Calculator:

image

Fixing SVG so that it is not clipped, Mortgage Affordability Calculator:

image

### CODE REVIEW

**Comment by tomas-stefano** [comment]
:+1:

File: app/assets/stylesheets/mortgage_calculator/components/_pie.scss [hunk 1]
```diff
@@ -6,3 +6,7 @@
     margin-left: 50px;
   }
 }
+
+.pie__svg {
+  min-height: 170px;
+}
```
Comment (munckymagik) [inline app/assets/stylesheets/mortgage_calculator/components/_pie.scss]: Do we still have a rule that all files should have a trailing newline?

File: app/assets/stylesheets/mortgage_calculator/components/_mortgagecalc.scss [hunk 2]
```diff
@@ -110,6 +116,7 @@
 }
 
 .mortgagecalc__spread {
+  margin-bottom: 0.5rem;
```
Comment (munckymagik) [inline app/assets/stylesheets/mortgage_calculator/components/_mortgagecalc.scss]: As this is vertical spacing should this be defined using `$baseline-unit` or does that not make sense in this case?

**Comment by munckymagik** [comment]
:+1:

File: app/assets/stylesheets/mortgage_calculator/components/_mortgagecalc.scss (hunk 2 above)
Comment (slaywell) [inline app/assets/stylesheets/mortgage_calculator/components/_mortgagecalc.scss]: The rest of the Mortgage Calculator doesn't use this yet but I think it's a good move, so I've changed it to use this variable now.

**Comment by munckymagik** [comment]
:+1: thanks Matt.

**Pull request merged**